pages/settings/profile.tsx

```diff
@@ -370,6 +374,23 @@ function SettingsView(props: ComponentProps<typeof Settings> & { localeProp: str
                 </div>
               </div>
             </div>
+            <div>
+              <label htmlFor="brandColor" className="block text-sm font-medium text-gray-700">
+                {t("brand_color")}
+              </label>
+              <div className="flex mt-1">
+                <input
+                  ref={brandColorRef}
+                  type="text"
+                  name="brandColor"
+                  id="brandColor"
+                  placeholder="#hex-code"
+                  className="block w-full px-3 py-2 mt-1 border border-gray-300 rounded-sm shadow-sm focus:outline-none focus:ring-neutral-500 focus:border-neutral-500 sm:text-sm"
+                  defaultValue={props.user.brandColor}
+                />
```

Comment:
Should we enhance the color picking experience? Maybe by using [react-color](https://casesandberg.github.io/react-color/) or enhancing the native [HTML5 color input](https://www.w3schools.com/TAGS/tryit.asp?filename=tryhtml5_input_type_color)? what are your thoughts @baileypumfleet ?

Reply:
That was planned as a follow up, but we can definitely encorporate that in this PR itself. I'm fine either way :)

Reply:
Yeah, we can follow up with @Jaibles about the best UX in here.

Reply:
I don't particularly mind, as long as we can find a dependency which is lightweight enough, or the HTML color input should work fine for most cases